### PR TITLE
docs: update talosctl install instructions for 1.8

### PR DESCRIPTION
**Pull request by rothgar**
Pulled changes from 1.7 docs

### CODE REVIEW

**Review by smira — APPROVED** [review]

File: website/content/v1.8/talos-guides/install/talosctl.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+```bash
+curl -sL https://talos.dev/install | sh
+```
+
+This script will work on macOS, Linux, and WSL on Windows.
+It supports amd64 and arm64 architecture.
+
+## Manual and Windows install
+
+All versions can be manually downloaded from the [talos releases page](https://github.com/siderolabs/talos/releases/) including Linux, macOs, and Windows.
```
Comment (themoriarti) [inline website/content/v1.8/talos-guides/install/talosctl.md:35]: macOs typo miss

File: website/content/v1.8/talos-guides/install/talosctl.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+---
+title: "talosctl"
+description: "Install talos cli client."
+---
+
+## Recommended
+
+The client can be installed and updated via the [Homebrew package manager](https://brew.sh/) for macOS and Linux.
```
[the remark this answers is not in the document]
Comment (rothgar) [reply website/content/v1.8/talos-guides/install/talosctl.md:8]: macOS 10.12 officially lower cases the 'm' They also dropped the redundant 'X' from the name

File: website/content/v1.8/talos-guides/install/talosctl.md (hunk 1 above)
Comment (rothgar) [reply website/content/v1.8/talos-guides/install/talosctl.md]: thanks for catching. That drives me crazy.

**Pull request merged**